build-system/tasks/e2e/describes-e2e.js

```diff
@@ -579,6 +611,35 @@ class EndToEndFixture {
       await controller.dispose();
     }
   }
+
+  /**
+   * Translate relative fixture specs into localhost test URL.
+   * @param {!TestSpec} spec
+   * @return {!TestSpec}
+   */
+  setTestUrl(spec) {
+    // TODO(rcebulko): See if it's possible to condense/reorg some test fixtures
+    // and have all e2e fixtures in one folder. Allowing this split for now so
+    // that, if fixtures under `manual` are moved, tests using normal e2e fixture
+    // directory don't need to be updated. Support for `testUrl` should
+    // eventually be removed entirely.
+    let {testUrl} = spec;
+    const {fixture, manualFixture} = spec;
```

Comment:
These two fixture folders should probably be deduped at some point, at which point `manualFixture` can be removed entirely. We should also consider entirely disallowing `testUrl`, as it will force test fixtures to stay in a centralized location `test/fixtures/e2e` and prevents accidentally introducing a test that makes a real network request.

Reply:
A quick CTRL+F of https://github.com/rcebulko/amphtml/compare/00ad4ef...rcebulko:vtest-updates shows that there are ~37 uses of the manual fixture directory in e2e tests. Around half of those are `base-carousel` fixtures. If we like this approach, all fixtures used in automated testing could be migrated out of `test/manual` into `test/fixtures/e2e`, and then all endtoend tests would specify a `fixture` directly and `test/manual` would contain only example HTML files for manual testing

Reply:
Update: Esther confirmed that `test/manual` existed before we did e2e testing, and any automated testing fixtures that ended up there should probably be moved. Created https://github.com/ampproject/amphtml/issues/32177 to track; will do that in a separate PR to keep the footprint small for this one.